docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/unified/index.md

```diff
@@ -0,0 +1,273 @@
+
+
+### Warehouse Specific
+
+<Tabs groupId="warehouse" queryString>
+<TabItem value="databricks" label="Databricks" default>
+
+| Variable Name        | Description                                                                                                                                                                                                                                                                                          | Default          |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------- |
+| `databricks_catalog` | The catalogue your atomic events table is in. Depending on the use case it should either be the catalog (for Unity Catalog users from databricks connector 1.1.1 onwards, defaulted to `hive_metastore`) or the same value as your `snowplow__atomic_schema` (unless changed it should be 'atomic'). | `hive_metastore` |
+
+</TabItem>
+<TabItem value="redshift/postgres" label="Redshift & Postgres">
+
+Redshift and Postgres use a [shredded](/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/transforming-enriched-data/index.md#shredded-data) approach for the context tables, so these variables are used to identify where they are, if different from the expected schema and table name. They must be passed in a stringified `source` function as the defaults below show.
+
+| Variable Name         | Default                                                |
+| --------------------- | ------------------------------------------------------ |
+| `page_view_context`   | `com_snowplowanalytics_snowplow_web_page_1`            |
+| `iab_context`         | `com_iab_snowplow_spiders_and_robots_1`                |
+| `ua_parser_context`   | `com_snowplowanalytics_snowplow_ua_parser_context_1`   |
+| `yauaa_context`       | `nl_basjes_yauaa_context_1`                            |
+| `consent_cmp_visible` | `com_snowplowanalytics_snowplow_cmp_visible_1`         |
+| `consent_preferences` | `com_snowplowanalytics_snowplow_consent_preferences_1` |
+| `consent_cmp_visible` |`com_snowplowanalytics_snowplow_cmp_visible_1`          |
+| `browser_context`     | `com_snowplowanalytics_snowplow_browser_context_1`     |
+| `session_context`     | `com_snowplowanalytics_snowplow_client_session_1`      |
+| `mobile_context`      | `com_snowplowanalytics_snowplow_mobile_context_1`      |
+| `geolocation_context` | `com_snowplowanalytics_snowplow_geolocation_context_1` |
+| `application_context` | `com_snowplowanalytics_mobile_application_1`           |
+| `screen_context`      | `com_snowplowanalytics_mobile_screen_1`                |
```

Comment:
I think some of these need updating now?

Reply:
also cwv was missing